src/mlpack/core/data/imputer.hpp

```diff
@@ -68,13 +68,13 @@ class Imputer
   //! Get the strategy
   const StrategyType& Strategy() const { return strategy; }
 
-  //! Modify the given given strategy (be careful!)
+  //! Modify the given strategy
   StrategyType& Strategy() { return strategy; }
 
   //! Get the mapper
   const MapperType& Mapper() const { return mapper; }
 
-  //! Modify the given mapper (be careful!)
+  //! Modify the given mapper
```

Comment:
Can you add a `.` to the end of the comment please so it is a complete sentence?

Reply:
Yes, I did that :)